src/main/java/org/folio/rest/impl/PatronNoticePoliciesAPI.java

```diff
@@ -23,9 +28,55 @@
 
   private static final Logger logger = LoggerFactory.getLogger(PatronNoticePoliciesAPI.class);
 
-  private static final String PATRON_NOTICE_POLICY_TABLE = "patron_notice_policy";
+  public static final String PATRON_NOTICE_POLICY_TABLE = "patron_notice_policy";
   public static final String STATUS_CODE_DUPLICATE_NAME = "duplicate.name";
 
+  @Override
+  public void getPatronNoticePolicyStoragePatronNoticePolicies(
+    int offset,
+    int limit,
+    String query,
+    Map<String, String> okapiHeaders,
+    Handler<AsyncResult<Response>> asyncResultHandler,
+    Context vertxContext) {
+
+    vertxContext.runOnContext(v -> {
+      try {
+        String tenantId = okapiHeaders.get(RestVerticle.OKAPI_HEADER_TENANT);
+        PostgresClient pgClient = PostgresClient.getInstance(vertxContext.owner(), tenantId);
+
+        String[] fieldList = {"*"};
+
+        CQL2PgJSON cql2pgJson = new CQL2PgJSON("patron_notice_policy.jsonb");
```

Comment:
Seems like there is constant value defined for table name

Reply:
replaced by `PATRON_NOTICE_POLICY_TABLE + ".jsonb"`